armory/scenarios/poisoning_gtsrb_scenario.py

```diff
@@ -224,18 +270,14 @@ def _evaluate(self, config: dict) -> dict:
                     poisoned_indices,
                 )
                 y_pred = classifier.predict(x_test)
-                test_metric.append(y_test, y_pred)
-
                 y_pred_targeted = y_pred[y_test == src_class]
                 if not len(y_pred_targeted):
                     continue
                 targeted_test_metric.append(
                     [tgt_class] * len(y_pred_targeted), y_pred_targeted
                 )
-            results["test_accuracy"] = test_metric.mean()
-            results["targeted_misclassification_accuracy"] = targeted_test_metric.mean()
-            logger.info(f"Test accuracy: {test_metric.mean():.2%}")
```

Comment:
We should keep test accuracy, as this should not be greatly impacted by targeted attacks (and is a strong detection signal if it is).